tests/unit/mixins/fastboot-compat-test.js

```diff
@@ -0,0 +1,12 @@
+import Ember from 'ember';
+import FastbootCompatMixin from 'ember-user-activity/mixins/fastboot-compat';
+import { module, test } from 'qunit';
+
+module('Unit | Mixin | fastboot compat');
+
+// Replace this with your real tests.
```

Comment:
You don't need the real fastboot service, you can simply inject a mock service called `fastboot` that has an `_isFastBoot` property. For the non-FastBoot test, you can literally just do nothing and check that the CP evaluates to false (undefined) as expected